app/lib/io/zip.js

```diff
@@ -0,0 +1,34 @@
+define([
```

Comment:
I'd like to change the module name to `zipUtil, because we still have a contender in mind that's a better fit to be just `zip` in the IO context.